readme.md

```diff
@@ -112,6 +112,24 @@ You can also supply a custom comparator function which will receive the followin
 - `sourceFile` *([Vinyl file object](https://github.com/wearefractal/vinyl#file))*
 - `destPath` *(string)* - Destination for `sourceFile` as an absolute path
 
+##### transform
+
+Type: `Function`
+
+Function to transform the path to the destination file.
+Should return the absolute path to the (renamed) destination file.
+
+Useful if you rename your file later on, like in the example below:
+
+```js
+gulp.task('marked', () =>
+	gulp.src('src/content/about.md')
+		.pipe(changed('app', {transform: path => path.replace('about.md', 'about/index.html')}))
```

Comment:
This is a bad example. Nobody should be using `.replace()` on a patch like this. Use the `path` methods.

Same with the below `rename()` usage.